### PR TITLE
Provide Run Selection button in Editor Title Menu

### LINKED ISSUE #2215: Provide F8 Action button on the menu in VS Code, just like the F8 in the PowerShell ISE Menu bar.
**Feedback Request:** Like in the PowerShell ISE, can you please provide a “Run Selection (F8)” button, right in the menu in VS code also (see below picture)? This will ease the transition of the workflow one uses in ISE, and enable more easy switch from ISE to Code. 
 
 
F8-Code

Thanks,
Girish.

### PR DESCRIPTION
**Pull request by jpogran**
## PR Summary

This adds a editor title menu button for the `powershell.RunSelection` command in the primary navigation group, and the `run` icon from the https://github.com/microsoft/vscode-icons repo.
This shows the button in the visible part of the menu when a PowerShell file is open.

This PR is marked as WIP because the icon chosen is the closest approximation I could find in the vscode suggested icon library, but if there is another one that is preferred I can change it to that.

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [X] PR has a meaningful title
- [X] Summarized changes
- [ ] PR has tests `NA`
- [ ] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

This PR fixes #2215

### CODE REVIEW

**Comment by TylerLeonhardt** [comment]
@jpogran can you use these svgs? 
[play.zip](https://github.com/PowerShell/vscode-powershell/files/3692642/play.zip)

They look like this:
image

I got some help from @isabelacmor mashing the vscode icons together 😅

**Comment by jpogran** [comment]
Updated with the new icon. Nice work 😄 

image

**Comment by TylerLeonhardt** [comment]
Looks good! Hmm I'm thinking it might be nice to have another button to the left of it that's just the play button, just like the ISE. That can just launch the debugger since that's what F5 does today.

@rjmholt @SydneyhSmith what do you think?

**Comment by rjmholt** [comment]
> what do you think?

Agreed!

**Comment by rjmholt** [comment]
I still need to checkout this branch and take a look at it, but it looks good so far!

**Comment by TylerLeonhardt** [comment]
@jpogran is that something you're interested in taking on? You can just use the play button you had previously and then just look up the command that it needs to execute to start debugging.

Let me know if you need help finding the command. I typically find it on the keybinding menu by hovering over the keybinding.

**Comment by jpogran** [comment]
Sure, will hook that up tomorrow

**Comment by jpogran** [comment]
@TylerLeonhardt I registered the command in the `Console.ts` file because I couldn't think of a better place to put it. I figured it made sense to be next to `Run Selection`. Let me know if you want the location changed

**Comment by SydneyhSmith** [comment]
@jpogran this is looking great to me, thanks for the work!

File: package.json [hunk 1]
```diff
@@ -161,10 +161,23 @@
         },
         "category": "PowerShell"
       },
+      {
+        "command": "PowerShell.Run",
```
Comment (TylerLeonhardt) [inline package.json]: @jpogran is there a reason this can't just be `workbench.action.debug.start`?

Comment (jpogran) [reply package.json]: No, I just had it in my head you need to make a command for anything you were going to use 😅 . Works fine as you suggest

**Comment by TylerLeonhardt** [comment]
ignore the macOS failure... I need to fix that.

**Comment by jpogran** [comment]
It's great to see the reaction to this, and the two approvals. Is there anything waiting on me to do to get this merged?

**Comment by TylerLeonhardt** [comment]
Nope! Thank you for your contribution! People are so pumped for this.

**Pull request merged**